docs/getting-started/security-ui.asciidoc

```diff
@@ -0,0 +1,108 @@
+
+TIP: All Elastic Security histograms, graphs, and tables contain an **Inspect** button so you can examine the {es} queries used to retrieve data throughout
+the app.
+
+Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timelines-ui,>> for more information.
+
+*Host and network events*
+
+View event and host counts specific to Elastic data shippers and apps, such as **Auditbeats** or **Elastic Endpoint Security**. Expand each category to view specific counts of hosts or network events related to the selected category.
+
+[role="screenshot"]
+image::images/events-count.png[]
+
+
+[float]
+[[hosts-ui]]
+=== Hosts page
+
+The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline. See <<hosts-overview>> for more information.
+
+[role="screenshot"]
+image::images/hosts-pg.png[]
+
+
+[float]
+[[network-ui]]
+=== Network page
+
+The Network page provides key network activity metrics via an interactive map and network event tables
+that enable interaction with the Timeline. See <<network-page-overview>> for more information.
+
+[role="screenshot"]
+image::images/network-ui.png[]
+
+
+[float]
+[[detection-engine-ui]]
+=== Detections page
+
+The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<ddetection-engine-overview>> for more information about detections, prebuilt rules, and alerts.
+
+[role="screenshot"]
+image::images/detections-ui.png[]
+
+[float]
+[[timelines-page]]
+=== Timelines page
+
+Use the Timelines page to investigate alerts and complex threats, such as lateral movement of malware across hosts
+in your network. Timelines are responsive and allow you to share your findings among other team members. See <<timelines-ui>> for information about getting started with Timelines.
+
+[role="screenshot"]
+image::images/timeline-ui.png[]
+
+[float]
+[[cases-ui]]
+=== Cases page
+
+The Cases page is used to open and track security issues directly in the {security-app}. For more information, see <<cases-overview>>.
+
+[role="screenshot"]
+image::images/cases-ui-home.png[]
+
+[float]
+[[admin-ui]]
+=== Administration page
+The Administration page allows you to view and manage hosts that are running Elastic Endpoint Security. You can also manage agent integration and monitor the configuration status of your hosts to ensure they are protected. See <<admin-pg-ov>> for more information.
```

Comment:
This linked page also fails for me, so I recommend omitting it until management/admin/admin-pg-ov.asciidoc is successfully included in the book:

```suggestion
The Administration page allows you to view and manage hosts that are running Elastic Endpoint Security. You can also manage agent integration and monitor the configuration status of your hosts to ensure they are protected.
// See <<admin-pg-ov>> for more information.
```